changelog.txt

```diff
@@ -1,6 +1,11 @@
 Version 1.13.2
 -------------
+Add cache update for authority migration. New token will always be written into the preferred cache location; For read, try preferred_cache first, followed by provided authority and aliased authority.
```

Comment:
nit: grammatically, the semicolon should not be followed by a capital letter. In this case, the semicolon could be replaced by a period and it would read fine